.github/ISSUE_TEMPLATE/module_wanted.yaml

```diff
@@ -0,0 +1,39 @@
+name: Module Wanted
+description: Request that a module be added to the registry
+title: "wanted: [github path of the module, e.g. bazelbuild/rules_foo]"
+labels: ["module wanted"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        The Central Registry is populated by community contributions.
+        You can file an issue to request that a volunteer add it to the registry.
```

Comment:
"request" sounds a bit strong here, even when combined with the following sentence.

Reply:
changed to

> You can file an issue to indicate that you depend on a module which isn't available here.
>        A volunteer may contribute that module, but there is no guarantee.
>        You may want to send a PR yourself, or consider sponsoring the work.